code/modules/shuttle/shuttle.dm

```diff
@@ -436,54 +453,46 @@
 	SSshuttle.mobile_docking_ports -= src
 
 /obj/docking_port/mobile/Destroy(force)
-	if(force)
-		unregister()
-		destination = null
-		previous = null
-		QDEL_NULL(assigned_transit) //don't need it where we're goin'!
-		shuttle_areas = null
-		remove_ripples()
-	. = ..()
+	unregister()
```

Comment:
why did you remove the `force` check here?

Reply:
If we're deleting it, we should clear everything, force or not. I would rather we fix any problem where it's somehow deleted without force where we don't want it to be, instead of keeping this snowflake check.